package.json

```diff
@@ -39,8 +39,6 @@
 	"dependencies": {
 		"@sindresorhus/is": "^1.0.0",
 		"@szmarczak/http-timer": "^3.0.0",
-		"@types/cacheable-request": "^6.0.1",
-		"@types/tough-cookie": "^2.3.5",
```

Comment:
I think these needs to be dependencies as they're used in the actual code.

Reply:
https://stackoverflow.com/a/45176764

Reply:
Only TypeScript users need this.

Reply:
> Only TypeScript users need this.

Do you think I should include them in dependencies since our exported type definitions includes them?

Reply:
I think that we shouldn't include the types as dependencies. They should be in devDependencies, because people that don't use TS will have to download 1.5MB+ data. People that want to use TS would need to `npm install @types/... @types/... @types/...`.

Reply:
Actually, I was not entirely correct. `@types/tough-cookie` can be a dev dependency as we don't directly depend on it, and it's up to the user to add it if they use `tough-cookie`. However, we do depend on `cacheable-request`, and we use its types, so I think we need to have `@types/cacheable-request` as a dependency, otherwise Got will not run when published and used in a TS project, right? Unless I'm missing something.

Reply:
Many people don't use TS and for them it's only bloat. NPM should somehow detect typescript projects and install types automatically...

Reply:
> Actually, I was not entirely correct. `@types/tough-cookie` can be a dev dependency as we don't directly depend on it, and it's up to the user to add it if they use `tough-cookie`. However, we do depend on `cacheable-request`, and we use its types, so I think we need to have `@types/cacheable-request` as a dependency, otherwise Got will not run when published and used in a TS project, right? Unless I'm missing something.

I am going to do this. I think both of you are correct, but maintaining out-of-the-box working status is important for TS users too, so the bloat might be inevitable ...